common/features/org.jboss.tools.common.all.feature/pom.xml

```diff
@@ -8,5 +8,6 @@
 	</parent>
 	<groupId>org.jboss.tools.common.features</groupId>
 	<artifactId>org.jboss.tools.common.all.feature</artifactId> 
+	<version>3.8.4-SNAPSHOT</version>
```

Comment:
This version is not required, it is basically code duplication when version is the same as in parent pom.xml. The same is true for all other duplicated versions below.